docs/user-guide/postinstall.md

```diff
@@ -124,3 +124,80 @@ To get a detailed log for Mysterium node service, you can use the following comm
 ```
 sudo journalctl -u mysterium-node.service
 ```
+
+## Built-in UI and Node LAN discovery
+
+It is possible to start a Mysterium node with a built-in UI server to do the basic configuration.
+
+The `--ui.enable` flag should be used to start a node with built-in UI.
```

Comment:
Changed this to be true by default in https://github.com/mysteriumnetwork/node/pull/1141
Either remove the section, or inverse it (provide a way to disable it via `--ui.enable=false`)

Reply:
Removed it.